src/test/java/org/bonitasoft/web/client/BonitaClientIT.java

```diff
@@ -105,7 +105,7 @@ class BonitaClientIT {
     private static final BonitaContainer<? extends BonitaContainer<?>> BONITA_CONTAINER = new BonitaContainer<>(
             BONITA_DOCKER_IMAGE);
 
-    private static final Semver _10_2 = new Semver("10.2", SemverType.LOOSE);
+    private static final Semver _10_2 = new Semver("10.2.0", SemverType.LOOSE);
```

Comment:
Can we set a "tolerance" mode when comparing versions, instead of setting the 3rd digit?

Reply:
I think it is mandatory, otherwise the orEqual part of the comparison don't work (see https://github.com/vdurmont/semver4j/issues/70)